### PR TITLE
add logging lines to send email job

### PR DESCRIPTION
**Pull request by GregorioDiStefano**
Signed-off-by: Gregorio Di Stefano <greg.distefano+github@gmail.com>
Changelog: none

### CODE REVIEW

**Comment by GregorioDiStefano** [comment]
@maciejmrowiec @0lmi

File: workers/send_email/worker/main.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 logging.basicConfig()
 log = logging.getLogger('conductor-workers.send_email')
-log.setLevel(logging.INFO)
```
Comment (maciejmrowiec) [inline workers/send_email/worker/main.py:11]: Any change we can make it configured from ENV ? (defaulting to INFO)

Comment (GregorioDiStefano) [reply workers/send_email/worker/main.py:11]: sure

**Comment by GregorioDiStefano** [comment]
@maciejmrowiec set DEBUG env. variable to enable debug logging

**Review by maciejmrowiec — APPROVED** [review]

**Pull request merged**